tests/js/spec/utils/usePrevious.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import {renderHook} from '@testing-library/react-hooks';
+
+import usePrevious from 'sentry/utils/usePrevious';
+
+describe('usePrevious', () => {
+  it('stores initial value', () => {
+    const {result} = renderHook(() => usePrevious('Initial Value'));
+    expect(result.current).toBe('Initial Value');
+  });
+
+  it('provides initial value', () => {
+    const {result} = renderHook(value => usePrevious(value), {
+      initialProps: 'Initial Value',
+    });
+
+    expect(result.current).toBe('Initial Value');
+  });
+
+  it('provides previous value', () => {
+    const {result, rerender} = renderHook(value => usePrevious(value), {
```

Comment:
thanks for creating tests for this hook 🎉  

Do you think you can convert the test to typescript? I believe only this line will need to be updated

Reply:
Oh yeah, idk why I even used jsx 🤦🏼